src/libaktualizr/uptane/fetcher_test.cc

```diff
@@ -149,10 +157,18 @@ int main(int argc, char** argv) {
 
   std::string port = TestUtils::getFreePort();
   server += port;
-  TestHelperProcess server_process("tests/fake_http_server/fake_http_server.py", port);
+  TestHelperProcess http_server_process("tests/fake_http_server/fake_http_server.py", port);
+  std::string treehub_port = TestUtils::getFreePort();
+  treehub_server += treehub_port;
+  TestHelperProcess ostree_server_process("tests/sota_tools/treehub_server.py", treehub_port);
 
+  TemporaryDirectory temp_dir;
+  int r = system((std::string("cp -r ") + argv[1] + std::string(" ") + temp_dir.PathString()).c_str());
+  if (r != 0) {
+    return -1;
+  }
+  sysroot = (temp_dir.Path() / "ostree_repo").string();
   sleep(3);
-
```

Comment:
You will need to rebase; we've improved this logic so that now we can wait for the server to be ready automatically instead of mandating 3 seconds.